collector/spec/collector.spec

```diff
@@ -28,7 +28,9 @@ Can also be used with InfluxDB and Grafana.
 %build
 # this command invokes the root Makefile of cmonitor repo, from inside the source tarball
 # produced by COPR that will pass all the options listed here to collector/Makefile
-%make_build DISABLE_UNIT_TESTS_BUILD=1 DISABLE_BENCHMARKS_BUILD=1 FMTLIB_MAJOR_VER=6 CMONITOR_LAST_COMMIT_HASH=__LAST_COMMIT_HASH__
+#pip3 install conan
```

Comment:
Strangely the %build here is not getting invoked here by the .copr/Makefile ...!!! 
Ideally it should install the conan and prometheus-cpp lib here and set the PROMETHEUS_SUPPORT flag to 1.